README.adoc

```diff
@@ -17,6 +17,8 @@
 :page-related-guides: ['microprofile-config', 'microprofile-metrics', 'cdi-intro', 'rest-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
+:page-seo-title: Checking the health of microservices using MicroProfile Health Check
+:page-seo-description: A tutorial on how to report the health status of Java microservices by implementing readiness and liveness checks using MicroProfile Health Check.
```

Comment:
Change
"A tutorial on how to report the health status of Java microservices by implementing readiness and liveness checks using MicroProfile Health Check."
To
"A tutorial on how to report the health status of Java microservices by implementing readiness and liveness checks by using MicroProfile Health Check."